src/memory_raw.c

```diff
@@ -195,3 +195,20 @@ static int memory_is_range_type_available(int type) {
         return 0;
     }
 }
+
+int memory_freemap_walk() {
```

Comment:
forgot about this. I used it for figuring out which line caused the "memory all allocated error"